voc/python/blocks.py

```diff
@@ -204,7 +204,7 @@ def add_callable(self, function, closure=False):
                 JavaOpcodes.DUP(),
         )
 
-        self.add_str(function.name)
+        self.add_str(function.code.co_name)
```

Comment:
@freakboy3742 for some reason, `function.name` ends up being "invoke" for closure functions, `function.code.co_name` seemed to work better.
I'm not completely sure what I'm doing here though, does this look okay?